convert_keys_yml_to_json.py

```diff
@@ -0,0 +1,11 @@
+import json
```

Comment:
What's this file for? Don't see it referenced anywhere.

Reply:
line 11